src/components/Footer/Footer.module.css

```diff
@@ -1,3 +1,98 @@
 .root {
     margin-top: auto;
+
+}
+
+.container {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+}
+
+.logoMobile {
+    @media (min-width: 640px) {
+        display: none;
+    }
+}
+
+.logo {
+    display: none;
+    @media (min-width: 640px) {
+        display: inline-flex;
+    }
+}
+
+
+.footerNavLinksWrapper {
+    display: inline-flex;
+    margin-left: auto;
+    padding-top: 24px;
+    padding-bottom: 24px;
+
+    @media (min-width: 840px) {
+        padding-top: 48px;
+        padding-bottom: 48px;
```

Comment:
Same comment as previously: I'd suggest using a base spacing from config.

Reply:
don't think it makes sense in this project